storage/database/dynamodb.go

```diff
@@ -0,0 +1,479 @@
+package database
+
+import (
+	"bytes"
+	"fmt"
+	"strings"
+	"sync"
+	"time"
+
+	"github.com/rcrowley/go-metrics"
+
+	"github.com/pkg/errors"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/dynamodb"
+	"github.com/aws/aws-sdk-go/service/dynamodb/dynamodbattribute"
+	"github.com/klaytn/klaytn/log"
+)
+
+var dataNotFoundErr = errors.New("data is not found with the given key")
+var nilDynamoConfigErr = errors.New("attempt to create DynamoDB with nil configuration")
+
+const dynamoWriteSizeLimit = 400*1024 - 32 // The maximum item size is 400 KB including a key size
+const dynamoBatchSize = 25
+const dynamoMaxRetry = 5
+
+// dynamo table provisioned setting
+const dynamoReadCapacityUnits = 10000
+const dynamoWriteCapacityUnits = 10000
```

Comment:
@winnie-byun Will you parameterize them? right?

Reply:
Yes I will :)